packages/react-components/src/organisms/EngagementTable.tsx

```diff
@@ -0,0 +1,133 @@
+import { borderRadius } from '../card';
+import { charcoal, neutral200, steel } from '../colors';
+import { rem, tabletScreen } from '../pixels';
+
+const container = css({
+  display: 'grid',
+  paddingTop: rem(32),
+});
+
+const gridTitleStyles = css({
+  display: 'none',
+  [`@media (min-width: ${tabletScreen.min}px)`]: {
+    display: 'inherit',
+    paddingBottom: rem(16),
+  },
+});
+
+const rowTitleStyles = css({
+  paddingTop: rem(32),
+  paddingBottom: rem(16),
+  ':first-of-type': { paddingTop: 0 },
+  [`@media (min-width: ${tabletScreen.min}px)`]: { display: 'none' },
+});
+
+const rowStyles = css({
+  display: 'grid',
+  padding: `${rem(20)} ${rem(24)} 0`,
+  borderBottom: `1px solid ${steel.rgb}`,
+  ':first-of-type': {
+    borderBottom: 'none',
+  },
+  ':nth-of-type(2n+3)': {
+    background: neutral200.rgb,
+  },
+  ':last-child': {
+    borderBottom: 'none',
+    marginBottom: 0,
+    paddingBottom: rem(15),
+    borderRadius: rem(borderRadius),
+  },
+  [`@media (min-width: ${tabletScreen.min}px)`]: {
+    gridTemplateColumns: '1fr 1fr 1fr 1fr 1fr 1fr',
+    columnGap: rem(15),
+    paddingTop: 0,
+    paddingBottom: 0,
+    borderBottom: `1px solid ${steel.rgb}`,
+  },
+});
+
+const titleStyles = css({
+  display: 'flex',
+  alignItems: 'center',
+  fontWeight: 'bold',
+  color: charcoal.rgb,
+  gap: rem(8),
+});
+
+const rowValueStyles = css({
+  display: 'flex',
+  gap: rem(6),
+  fontWeight: 400,
+});
+
+const iconStyles = css({
+  display: 'flex',
+  gap: rem(3),
+});
+
+export type EngagementData = {
+  id: string;
+  name: string;
+  isInactive: boolean;
+  memberCount: number;
+  presentationCount: number;
+  totalSpeakerCount: number;
+  uniqueAllRolesCount: number;
+  uniqueKeyPersonnelCount: number;
+};
+type EngagementTableProps = {
+  data: EngagementData[];
+};
+
+const EngagementTable: React.FC<EngagementTableProps> = ({ data }) => (
+  <>
+    <Card padding={false}>
+      <div css={container}>
+        <div css={[rowStyles, gridTitleStyles]}>
+          <span css={titleStyles}>Team</span>
+          <span css={titleStyles}>Members</span>
+          <span css={titleStyles}>Presentations</span>
+          <span css={titleStyles}>Total Speakers</span>
+          <span css={titleStyles}>Unique Speakers: All Roles</span>
+          <span css={titleStyles}>Unique Speakers: Key Personnel</span>
+        </div>
+        {data.map((row) => (
+          <div key={row.id} css={[rowStyles]}>
+            <span css={[titleStyles, rowTitleStyles]}>Team</span>
+            <p css={iconStyles}>
+              <Link href={network({}).teams({}).team({ teamId: row.id }).$}>
+                {row.name}
+              </Link>
+
+              {row.isInactive && <InactiveBadgeIcon />}
+            </p>
+            <span css={[titleStyles, rowTitleStyles]}>Members</span>
+            <p css={rowValueStyles}>{row.memberCount} </p>
+            <span css={[titleStyles, rowTitleStyles]}>Presentations</span>
```

Comment:
It seems it's called "Events" on Figma

![Screenshot 2024-07-08 at 13 47 47](https://github.com/yldio/asap-hub/assets/16595804/83874c85-44f7-44bf-9541-118d24d0c329)



```suggestion
            <span css={[titleStyles, rowTitleStyles]}>Events</span>
```